src/pg/pg-store.ts

```diff
@@ -621,6 +630,36 @@ export class PgStore extends BasePgStore {
     return inscription_id;
   }
 
+  private async insertInscriptionCount(args: { block_height: number }): Promise<void> {
+    await this.sql`
+      WITH previous AS (
+        SELECT COALESCE((
+          SELECT inscriptions_total
+          FROM inscriptions_per_block
+          WHERE block_height = ${args.block_height - 1}
```

Comment:
There could be a block with 0 inscriptions that breaks this calculation. You should select the highest available (but lower than the given height) block height instead